### PR TITLE
Update as per changes to integro-backend.

### PR DESCRIPTION
**Pull request by andre-at-situ**
## Description

Added a number of small changes based on tests run in Integro backend. Might have to version bump again.

## Type of change

-   [x] Bug fix (non-breaking change which fixes a bug)
-   [x] Chore (non-breaking change that refactors, tidies up or formats code)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Check-in for breaking change (if applicable)

-   [x] Those packages/repos that depend on this repo have been updated for the breaking change.

## How Has This Been Tested?

Tests have been run successfully in integro-backend.

## Checklist:

-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: entegywrapper/content/categories.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def create_categories(
     ----------
         `template_type` (`TemplateType`): the templateType of the page holding the categories
 
-        `categories` (`list[Category]`): the categories to create
+        `categories` (`list[Union[Category, ContentCreate]]`): the categories to create
```
Comment (willsawyerrrr) [inline entegywrapper/content/categories.py:201]:
```suggestion
        `categories` (`list[Category | ContentCreate]]`): the categories to create
```

Prefer `|` over `Union`. This is also consistent with the syntax in the method signature.

File: entegywrapper/content/categories.py [hunk 2]
```diff
@@ -372,7 +378,7 @@ def delete_categories(
     else:
         raise ValueError("Please specify an identifier")
 
-    response = self.delete(self.api_endpoint + "/v2/Categories/Delete", data=data)
+    response = self.post(self.api_endpoint + "/v2/Categories/Delete", data=data)
```
Comment (willsawyerrrr) [inline entegywrapper/content/categories.py:381]: [Docs](https://situ.entegysuite.com.au/Docs/Api/content-delete) suggest this should use `DELETE`.

**Review by willsawyerrrr — CHANGES_REQUESTED** [review]

File: entegywrapper/content/categories.py (hunk 2 above)
Comment (andre-at-situ) [reply entegywrapper/content/categories.py:381]: Aha.. they do.. but that doesn't work hence the change... thanks for your accurate API doc updates Entegy!

This function is under test now as well. So if they ever change it to what its meant to be we'll know.

File: entegywrapper/content/categories.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Optional
+from typing import TYPE_CHECKING, Any, Optional, Union
```
Comment (willsawyerrrr) [inline entegywrapper/content/categories.py:3]: This import is no longer needed.

Approved pending its removal.

Comment (andre-at-situ) [reply entegywrapper/content/categories.py]: Done.

**Pull request merged**